ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -712,6 +712,23 @@ MAU_SFSS2X_SFVSC2X_CHWSC_HWSC_SSPC_VOADM_FDPM_ZSPM_OAFM:
   opt_uses:
   - smoke_alarm
 
+MAU_DFSS_DFVSC_DSP_CHWDC_HWDC_OADM_ZSPM_OA:
+  guid: "9ba6dc9b-1b5b-4f05-8be2-2ff4b0bb7390"
+  description: "Single zone heating and cooling make-up air unit with zone static pressure control and outside air temperature monitoring."
+  is_canonical: true
+  implements:
+  - MAU
+  - DFSS
+  - DFVSC
+  - DSP
+  - CHWDC
+  - HWDC
+  - OADM
+  - ZSPM
+  - OA
```

Comment:
Seems odd to have `OA` on this equipment, typically we only use `OA` to model stand-alone weather stations